pkgs/tools/misc/sp800-90bEntropyAssessment/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bzip2
+, libdivsufsort
+, jsoncpp
+, openssl
+, mpfr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sp800-90bEntropyAssessment";
+  version = "1.1.6";
+
+  src = fetchFromGitHub {
+    owner = "usnistgov";
+    repo = "SP800-90B_EntropyAssessment";
+    rev = "v${version}";
+    sha256 = "sha256-KZQ7kC0PbBkjLEQZIqYakQ91OvCxruhdfUwiRHtno3w=";
+  };
+
+  buildInputs = [ bzip2 libdivsufsort jsoncpp openssl mpfr ];
+
+  sourceRoot = "source/cpp";
+
+  aarch64Flags = lib.optionals stdenv.hostPlatform.isAarch64 [
+    "ARCH=aarch64"
+    "CROSS_COMPILE=aarch64-linux-gnu-"
+  ];
+
+  makeFlags = lib.optionalString stdenv.hostPlatform.isAarch64 "${lib.strings.concatStringsSep " " aarch64Flags}";
```

Comment:
This construction looks a little weird, why not just add the required flags directly to `makeFlags`? I also think you don't need to concatenate the flags yourself.

Reply:
This is now changed as you suggested.